packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -553,7 +553,7 @@ export interface CallingTheme {
 
 // @public
 export type CallParticipantListParticipant = ParticipantListParticipant & {
-    state: 'Idle' | 'Connecting' | 'Ringing' | 'Connected' | 'Hold' | 'InLobby' | 'EarlyMedia' | 'Disconnected';
+    state: ParticipantState;
```

Comment:
(non-blocking comment, and I think this is mentioned in another PR somewhere so might be a duplicate comment)
but I have reservervations around the word `state` here - just because its used in so many contexts throughout our codebase and API, and things like isMuted and isScreenSharing are also the state of a participant. Alternate suggestions: `callConnectionState`, `connectionState` (grr still having to use state as a suffix lol - struggling to think of anything different)

Reply:
The `state` attribute here is part of our public api. Can't change it now.
I didn't add it.